src/database/hooks/useSaveAnonChatHook.ts

```diff
@@ -104,7 +104,8 @@ const useSaveAnonChatHook = () => {
       console.log(e);
     } finally {
       refresh('channelList');
-      refresh('chat');
+      // refresh('chat');
+      refreshWithId('chat', initAnonymousChat.message.cid);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Replacing the general `refresh('chat')` with `refreshWithId('chat', initAnonymousChat.message.cid)` is a good improvement. This change will only refresh the chat with the specified ID instead of refreshing all chats, which can improve performance and efficiency.
```diff
-      refresh('chat');
+      refreshWithId('chat', initAnonymousChat.message.cid);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->